src/plugins/chat.ts

```
@@ -2315,7 +2343,44 @@ export async function apply(ctx: Context, config: Config) {
                 nextReplyReasons
             )
 
-            service.muteAtLeast(session, copyOfConfig.coolDownTime * 1000)
+            await service.muteAtLeast(
+                session,
+                copyOfConfig.coolDownTime * 1000
+            )
```

ESLint

[error] Replace `⏎················session,⏎················copyOfConfig.coolDownTime·*·1000⏎············` with `session,·copyOfConfig.coolDownTime·*·1000`

(prettier/prettier)